utils/model/build_core.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+"""
+This file provides the code generator which transforms gaphor/UML/uml2.gaphor
+into gaphor/UML/uml2.py.
+
+Also a distutils tool, build_uml, is provided.
+"""
+
+import os.path
+from distutils.core import Command
+from distutils.dep_util import newer
+from distutils.dir_util import mkpath
+from distutils.util import byte_compile
+
+from utils.model import gen_uml
+
+
+def generate_uml2(force=False):
```

Comment:
Should we rename this to generate_core?

Reply:
Actually I want to have a proper script, with command line arguments that I can use instead. To avoid confusion I'll change this, though ;)